tests/unit/test_user.py

```diff
@@ -110,5 +111,62 @@ def test_signout(self):
         response = self.app.get('/signout', follow_redirects=True)
         self.assertEquals(response.status_code, 200)
 
+    #Test modify profile
+    def test_modify_profile(self):
+        create_user(self.user)
+        db_session = db.Session()
+        user_query_res = db_session.query(User).filter(User.username == test_cfg.USER_NAME).first()
```

Comment:
I think we don't need to get the user id from database.
We can use "remove_user_by_username" function to delete user from database